src/test/java/com/gooddata/dataload/processes/DataloadProcessesTest.java

```diff
@@ -6,22 +6,33 @@
 package com.gooddata.dataload.processes;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.hamcrest.CoreMatchers;
 import org.testng.annotations.Test;
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
 
+import java.util.regex.Pattern;
+
 public class DataloadProcessesTest {
 
+    private DataloadProcesses processes;
+
     @Test
     public void testDeserialization() throws Exception {
-        final DataloadProcesses processes = new ObjectMapper()
+        processes = new ObjectMapper()
                 .readValue(getClass().getResourceAsStream("/dataload/processes/processes.json"), DataloadProcesses.class);
 
         assertThat(processes, is(notNullValue()));
         assertThat(processes.getItems(), hasSize(1));
     }
 
+    @Test(dependsOnMethods = "testDeserialization")
```

Comment:
well, some of these domain objects are only created from deserialization and have no public constructor (most of them has private constructor). I rather have this test dependency, then turn all @JsonCreator constructors visibility to package private and using them manually from tests.